Tasks/CloudFormationDeleteStack/task.json

```diff
@@ -2,18 +2,13 @@
     "id": "089bda61-367d-4d7c-a06b-88f7e2f20f22",
     "name": "CloudFormationDeleteStack",
     "friendlyName": "AWS CloudFormation Delete Stack",
-    "description": "Deletes an AWS CloudFormation stack.",
+    "description": "Deletes a AWS CloudFormation stack.",
```

Comment:
The changes to "a AWS" in this PR seem odd to me.

Reply:
I got a little to excited about a vs an, reverted this change